layouts/partials/tags.html

```diff
@@ -0,0 +1,14 @@
+<hr>
+{{ if isset .Site.Taxonomies "tags" }}
+{{ if not (eq (len .Site.Taxonomies.tags) 0) }}
+<div class="tagslist">
+  <p class="asidetitle">{{ .Site.Params.Strings.Tags | default "Tag Cloud" }}</p>
+  <ul class="clearfix">
+    {{ range $name, $items := .Site.Taxonomies.tags }}
+	<li><a href="{{ $.Site.BaseURL }}/tags/{{ $name | urlize | lower }}" title="{{ $name }}">{{ .Page.Title }}<sup>{{ len $items }}</sup></a></li>
```

Comment:
using `{{ .Page.Permalink }}` for the tag url instead of manually constructing it would seem cleaner here, unless i'm missing something

Reply:
Im going to mark this as resolved based on marcelbaumann's pr that I've merged into my branch.